docs/pcawg/data.md

```diff
@@ -175,3 +175,25 @@ _aws_ cli commands, one line per file. One line may look like this:
 `aws --profile pdc --endpoint-url https://bionimbus-objstore-cs.opensciencedatacloud.org s3 cp s3://pcawg-tcga-brca-us/f99f7e36-8b6f-5cf0-854f-4b832d5962a4 .`
 
 You may execute the above line directly on the command line to download the object to local.
+
+# Analyze PCAWG Data on Collaboratory
+
+### Leverage compute and data colocation
+
+As noted above, [a large portion of the PCAWG data](https://icgc.org/447) is hosted at
+the [Cancer Genome Collaboratory](https://cancercollaboratory.org) - an
+OpenStack based academic cloud computing platform. This makes Collaboratory an ideal choice to perform your own analyses
+on PCAWG data, it's so-called <i>move compute to data</i> approach. In essence, there is no need to download
```

Comment:
would suggest here:

"based on the <i>move compute to data</i> approach."

Reply:
changed to: "following a commonly practised _move compute to data_ approach"